src/components/Inputs/ZipInput/ZipInput.js

```diff
@@ -58,9 +58,7 @@ export const ZipInput = (props) => {
       <InputLabel name={name} labelCopy={labelCopy} allCaps={allCaps} />
       <MaskedInput
         mask={[/\d/, /\d/, /\d/, /\d/, /\d/]}
-        type="text"
-        inputMode="numeric"
-        pattern="[0-9]*"
+        type="tel"
```

Comment:
We'll want to test these on the CMS netlify preview side on our mobile phones. But, in theory, if it's working like our current shared/fields are, this should "just work" cc / @zach-ethos